modules/S05_firmware_details.sh

```diff
@@ -20,7 +20,7 @@
 
 S05_firmware_details()
 {
-  module_log_init "firmware_testing_details"
+  module_log_init "S05_firmware_testing_details"
```

Comment:
cosmetic update.
Why not using:
module_log_init "${FUNCNAME[0]}"

Reply:
Why I didn't use it: I am a bad programmer ;) Sounds reasonable and I will implement it and also split this pr in two different pr's.